File: 260128/상하로 반복하며 출력/print-in-up-down-reverse.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);

        int n=sc.nextInt();
        int[][] answer=new int[n][n];

        for(int i=0; i<n; i++){
            for(int j=0; j<n; j++){
                if(i%2!=0){
                    answer[j][i]=n-j;
                }
                else{
                    answer[j][i]=j+1;
                }
            }

        }

        for(int i=0; i<n; i++){
            for(int j=0; j<n; j++){
                System.out.printf("%d", answer[i][j]);
            }
            System.out.println();
        }
    }
}

File: 260303/상하로 반복하며 출력/print-in-up-down-reverse.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int n;
    cin>>n;

    int cnt=n;
    
    for(int i=1; i<n+1; i++){
        for(int j=0; j<n; j++){
            if(j%2==0){
                cout<<i;
            }
            else cout<<cnt;
        }
        cnt--;
        cout<<endl;
    }
    return 0;
}